multiregion/README.md

```diff
@@ -278,10 +278,18 @@ Stop the demo and all Docker containers.
 
 ## Troubleshooting
 
-1. If containers fail to ping each other (e.g., failures seen in running `./scripts/validate_connectivity.sh`), stop the demo, clean up the Docker environment, and then start the demo again:
+1. If containers fail to ping each other (e.g., failures seen in running `./scripts/validate_connectivity.sh`), then stop the demo, clean up the Docker environment, and restart the demo:
 
 ```
+# Stop demo
 ./scripts/stop.sh
-docker container stop $(docker container ls -a -q) ; docker container rm $(docker container ls -a -q) ; docker volume prune -f
+
+# Clean up the Docker environment
+for c in $(docker container ls -q --filter "name=pumba"); do docker container stop "$c" && docker container rm "$c"; done
+docker-compose down -v --remove-orphans
+for v in $(docker volume ls -q --filter="dangling=true"); do docker volume rm "$v"; done
+docker volume prune -f
```

Comment:
If I understand correctly, the second line will remove any volume that is dangling and the third line will remove all volumes without any confirmation (forcefully).

Unless containers started outside of compose are creating volumes, I _think_ the `docker-compose down -v` in line 1 should be sufficient, and will only remove volumes related to this compose project instead of docker system wide.  If the `pumba` or other containers create volumes, then maybe we should name & remove them explicitly?